src/components/LoginForm/index.js

```diff
@@ -9,15 +9,32 @@ import useForm from 'hooks/useForm';
 import useValidation from 'hooks/useValidation';
 import loginValidations from 'validations/loginValidations';
 import ErrorView from 'components/common/ErrorView';
+import Button from 'components/common/Button';
 import useTextInputProps from 'hooks/useTextInputProps';
 import strings from 'locale';
+import { BLACK, ERROR } from 'constants/colors';
 import styles from './styles';
 
 const FIELDS = {
   email: 'email',
   password: 'password',
 };
 
+const BUTTONPROPS = {
```

Comment:
You can add these styles to the file `styles.js` and import from there, to keep consistent and have all the component styles on the same place 
Also, the names should follow camelCase as the rest of the styles 😄

Reply:
In addition to this is important to have the styles in a stylesheet not a simple object. This is because the stylesheet behind the scenes saves the styles natively. if the styles are created as an object they travel through the bridge on every re-render wasting bandwidth and reducing performance overall